.github/workflows/publish-binary.yml

```diff
@@ -109,7 +111,7 @@ jobs:
     strategy:
       matrix:
         cpu: ["", "skylake", "znver3"]
-        node: ["tanssi-node", "container-chain-frontier-node", "container-chain-simple-node"]
+        node: ["tanssi-node", "tanssi-relay", "container-chain-frontier-node", "container-chain-simple-node"]
```

Comment:
same thing here, I think the best would be that we separate this in another job

Reply:
Update at https://github.com/moondance-labs/tanssi/pull/696/commits/c5f31b35858312bf5ea3eda86b63def77f53400b